docs/getting-started/harper-concepts.md

```diff
@@ -0,0 +1,38 @@
+# Harper Concepts
+
+As you begin your journey with Harper, there are a few concepts and definitions that you should understand.
+
+## Components
+Harper components are a core Harper concept defined as flexible JavaScript based extensions of the highly extensible core Harper platform. They are executed by Harper directly and have complete access to the Harper [Global APIs](https://docs.harperdb.io/docs/technical-details/reference/globals) (such as Resource, databases, and tables).
+
+A key aspect to components are their extensibility; components can be built on other components. For example, a [Harper Application](https://docs.harperdb.io/docs/developers/applications) is a component that uses many other components. The [application template](https://github.com/HarperDB/application-template) demonstrates many of Harper's built-in components such as [rest](https://docs.harperdb.io/docs/developers/components/built-in#rest) (for automatic REST endpoint generation), graphqlSchema (for table schema definitions), and many more.
+
+## Applications
+Applications are a subset of components that cannot be used directly and must depend on other extensions. Examples include defining schemas (using [graphqlSchema](https://docs.harperdb.io/docs/developers/components/built-in#graphqlschema) built-in extension), defining custom resources (using jsResource built-in extension), hosting static files (using static built-in extension), enabling REST querying of resources (using rest built-in extension), and running Next.js, Astro, or Apollo applications through their respective extensions.
```

Comment:
missing links for `jsResource`, `static`, and `rest` built-ins, as well as `next.js`, `astro`, and `apollo` custom extensions